src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -428,12 +439,11 @@ internal static bool IsEdgeSupportedType(Type type)
                 type = type.GetElementType();
             }
 
-            if (type == typeof(int) ||
-                type == typeof(double) ||
+            if (type.IsPrimitive ||
```

Comment:
Yes it will still be a string - however with some testing I learned that Edge will ToString the DateTime itself, so I just pass it through. I did some sandbox testing and verified the types that Edge supports. There was a set of primitives that we were missing.

I had started with just adding explicit handling for DateTime. long, and some of the other new types used by EventHub that weren't supported, and decided that IsPrimitive is better than special casing each.